test/user-pagination.test.ts

```diff
@@ -0,0 +1,68 @@
+import { describe, it } from 'mocha';
+import { expect } from 'chai';
+import chaiExclude from 'chai-exclude';
+import { getValidToken, makeRequest, userRepository } from 'test/helpers';
+import { usersQuery } from 'test/graphql-snippets';
+import { saveFakeUsers } from 'scripts/seed';
+
+describe('users query', () => {
+  afterEach(async () => {
+    await userRepository.delete({});
+  });
+
+  it('should return the specified number of users sorted by name', async () => {
+    const token = getValidToken();
+    const storedUsers = await saveFakeUsers(20);
+
+    const res = (await makeRequest({ query: usersQuery, variables: { count: 15, offset: 0 }, token })).data;
+
+    const storedUsersSimplified = storedUsers.map((u) => {
```

Comment:
```suggestion
    const storedUsersSimplified = storedUsers.map((user) => {
```

É legal evitar abreviações, prezando sempre pela clareza do código.